Dockerfile

```diff
@@ -36,9 +36,6 @@ EXPOSE 3030 24567
 
 COPY scripts/run_docker.sh /usr/local/bin/run.sh
 
-# Remove the files to keep docker smaller.
-RUN rm -rf /near
```

Comment:
since this doesn't reduce size, i keep it for doing experiment inside. e.g. exec into, clone near-runtime-ts and run integration tests. we can also consider release both nearcore-dev (has source code and built) and nearcore (just binary) docker images.